buildout.cfg

```diff
@@ -9,7 +9,7 @@ python-gettext = >= 3
 zc.recipe.testrunner = >= 2
 zope.annotation = >= 4
 zope.browser = >= 2
-zope.component = >= 4
+zope.component = <= 4.2.1 , >= 4
```

Comment:
It seems to do the trick, cool!
If this is a hard dependency for the code it should also be in `setup.py`. Additionally a new issue for fixing this dependency would be nice. zope.component==4.2.1 is nearly 5 years old!